web/impact/impact/v1/views/user_history_view.py

```diff
@@ -22,4 +23,5 @@ class UserHistoryView(BaseHistoryView):
                      UserBecameConfirmedMentorEvent,
                      UserBecameFinalistEvent,
                      UserCreatedEvent,
-                     UserJoinedStartupEvent]
```

Comment:
I was seeing this odd result when looking at the history api call (note the programs dont line up)
<img width="955" alt="screen shot 2017-09-26 at 1 30 30 pm" src="https://user-images.githubusercontent.com/196425/30874669-0a5de710-a2bf-11e7-9c19-5c9b040ec8a0.png">
not sure if this is a consequence of the data cleaning procedure or not

Reply:
Good catch!  It was using str() rather than .short_name().  Fixed.